code/modules/urist/items/clothes/uristclothes.dm

```diff
@@ -923,4 +923,24 @@ Update 26/07/2014 - All generic clothing goes under obj/item/clothing/under/uris
 	name = "facebook"
 	desc = "Did someone really tape a book to their face?"
 	icon_state = "facebook"
-	item_state = "facebook"
\ No newline at end of file
+	item_state = "facebook"
+
+//why is this not storage? Duplicate item, but it makes way more sense as a storage item.
+
+/obj/item/clothing/suit/captunic/capjacket
```

Comment:
I don't know why this is here. This was a separate thing. I think I fucked up some branches somewhere. I'll fix this.